demo/addons/fmod/Fmod.gd

```diff
@@ -125,13 +125,18 @@ func _init() -> void:
 	godot_fmod.connect("sound_played", self, "on_sound_played")
 	godot_fmod.connect("sound_stopped", self, "on_sound_stopped")
 	print("Fmod Gdnative interface managed by a GDScript wrapper")
-	
+
+
 func _notification(what):
 	if what == NOTIFICATION_PREDELETE:
 		if started:
 			shutdown()
 		godot_fmod.free()
 
+func _ready():
+	###SETUP FMOD###
+	load_configuration("res://addons/fmod/fmod_config.cfg", true)
```

Comment:
The configuration is part of the demo project and not the addon in itself, you have to handle the case where the configuration file is missing. 
After all, you have to generate it a first in the editor when setting up a new project.